File: lógica/JS/vetor/soma-vetores.js
// Problema "soma_vetores" 
// Faça um programa para ler dois vetores A e B, contendo N elementos cada. Em seguida, gere um terceiro vetor C onde cada elemento de C é a soma dos elementos correspondentes de A e B. Imprimaco vetor C gerado.

var N, vetA = [], vetB = [], vetC = [], soma;

N = parseInt(prompt('Quantos valores vão ter em cada vetor ?'));

for(let i = 0; i < N; i++) {
    vetA[i] = parseInt(prompt('Digite os valores do Vetor A:'));
}

for(let i = 0; i < N; i++) {
    vetB[i] = parseInt(prompt('Digite os valores do Vetor B:'));
}

console.log('Vetor Resultante:')
for(let i = 0; i < N; i++) {
   soma = vetA[i] + vetB[i];
   console.log(soma); 
}

File: lógica/Python/vetor/soma_vetores.py
# Problema "soma_vetores" 
# Faça um programa para ler dois vetores A e B, contendo N elementos cada. 
# Em seguida, gere um terceiro vetor C onde cada elemento de C é a soma dos elementos correspondentes de A e B. 
# Imprima o vetor C gerado.

n = int(input('Quantos valores vão ter cada vetor ? '))
vetA = [0 for x in range(n)]
vetB = [0 for x in range(n)]
vetC = [0 for x in range(n)]
soma: int

for i in range(n):
    vetA[i] = int(input('Digite os valores do vetor A: '))

for i in range(n):
    vetB[i] = int(input('Digite os valores do vetor B: '))

for i in range(n):
    vetC[i] = vetA[i] + vetB[i]

print("VETOR RESULTANTE:")
for i in range(n):
    print(f"{vetC[i]}")
